packages/angular/projects/ui-angular/src/lib/services/authenticator.service.ts

```diff
@@ -166,10 +166,12 @@ export class AuthenticatorService implements OnDestroy {
     };
   }
 
-  /** @deprecated For internal use only */
-  public subscribe(callback: AuthSubscriptionCallback) {
+  public subscribe(callback: AuthSubscriptionCallback): () => void {
     if (this._authService) {
-      return this._authService.subscribe(callback);
+      const subscription = this._authService.subscribe(() => {
+        callback({ ...this._facade, ...this._sendEventAliases });
+      });
+      return subscription.unsubscribe;
```

Comment:
return `unsubscribe` function directly, so customers can just do 

```ts
const unsubscribe = this.authenitcator.subscribe(...);
```

like we do above.

Reply:
Good point, and I see that this is much more [standard](https://github.com/aws-amplify/amplify-ui/pull/2935/files#diff-12783128521e452af0cfac94b99b8d250413c516ec71fe6d97dbea666ff7ba27R5-R7) in Angular ecosystem. I'll update this back to `unsubscribe()`!

Reply:
Done in [e7a32a1](https://github.com/aws-amplify/amplify-ui/pull/2817/commits/e7a32a177a34ef1d3ccb1be9145dbc8a314797dc)